scripts/bash/enceladus_env.template.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2018-2019 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Environment configuration
+MENAS_URI="http://localhost/menas/api"
```

Comment:
default out of the box config is port 8080, even on our servers. Maybe let the template be `...localhost:8080/menas/..`